### PR TITLE
Update Hugging Face 🤗 Integration Documentation

### PR DESCRIPTION
**Pull request by simoninithomas**
Based on #1128 I updated the documentation.

## Description

- Created a new tutorial colab that takes into account the new features such as `package_to_hub`. It is hosted on huggingface\huggingface_sb3 repo.
- Adding `package_to_hub` and `push_to_hub` in the documentation.

The only problem I have from now with the HTML version is that the link is not truncated:
image

## Motivation and Context

- This solves the issue #1128, the documentation was outdated.

- [ ] I have raised an issue to propose this change ([required](https://github.com/DLR-RM/stable-baselines3/blob/master/CONTRIBUTING.md) for new features and bug fixes)

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [X] Documentation (update in the documentation)

## Checklist:

- [X] I've read the [CONTRIBUTION](https://github.com/DLR-RM/stable-baselines3/blob/master/CONTRIBUTING.md) guide (**required**)
- [X] I have updated the changelog accordingly (**required**).
- [ ] My change requires a change to the documentation.
- [ ] I have updated the tests accordingly (*required for a bug fix or a new feature*).
- [ ] I have updated the documentation accordingly.
- [ ] I have reformatted the code using `make format` (**required**)
- [ ] I have checked the codestyle using `make check-codestyle` and `make lint` (**required**)
- [ ] I have ensured `make pytest` and `make type` both pass. (**required**)
- [ ] I have checked that the documentation builds using `make doc` (**required**)

Note: You can run most of the checks using `make commit-checks`.

Note: we are using a maximum length of 127 characters per line

### CODE REVIEW

**Review by araffin — APPROVED** [review]
LGTM, thanks =)

File: docs/guide/integrations.rst [hunk 1]
```diff
@@ -47,21 +47,20 @@ Hugging Face 🤗
...
 
-You can see the list of stable-baselines3 saved models here: https://huggingface.co/models?other=stable-baselines3
+You can see the list of stable-baselines3 saved models here: https://huggingface.co/models?library=stable-baselines3
 Most of them are available via the RL Zoo.
 
 Official pre-trained models are saved in the SB3 organization on the hub: https://huggingface.co/sb3
 
-We wrote a tutorial on how to use 🤗 Hub and Stable-Baselines3 here: https://colab.research.google.com/drive/1GI0WpThwRHbl-Fu2RHfczq6dci5GBDVE#scrollTo=q4cz-w9MdO7T
+We wrote a tutorial on how to use 🤗 Hub and Stable-Baselines3 
+here: https://colab.research.google.com/github/huggingface/huggingface_sb3/blob/main/notebooks/sb3_huggingface.ipynb
```
Comment (araffin) [inline docs/guide/integrations.rst:56]:
```suggestion
`here <https://colab.research.google.com/github/huggingface/huggingface_sb3/blob/main/notebooks/sb3_huggingface.ipynb>`_
```

**Review by araffin — CHANGES_REQUESTED** [review]
After giving it a second look, I noticed minor things that need to adjusted.
Could you give me access to your fork?
as you use master branch, it is protected and I cannot push my edits to it...

**Comment by simoninithomas** [comment]
My bad for the branch, yes I've just added you to the collaborators of the fork.

**Pull request merged**